rain_api_core/egress_util.py

```diff
@@ -150,16 +150,9 @@ def process_request(varargs, b_map):
     return path, bucket, object_name, headers
 
 
-def check_private_bucket(bucket, private_buckets, b_map):
+def check_private_bucket(bucket, b_map):
```

Comment:
OH, you updated the functions here to remove the `private_buckets` and `check_public_bucket ` variable, which is good, but they're still being called with 3 params from [app.py](https://github.com/asfadmin/thin-egress-app/blob/a0e091f5cd09759fedfe13803b12c87065101c29/lambda/app.py#L458)